e2e/playwright/package.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "plugin-e2e-tests",
+    "scripts": {
+        "test": "PW_SLOMO=200 npm run test --prefix ../../../mattermost/e2e-tests/playwright -- --project=chrome --config='../../../mattermost-plugin-todo/e2e/playwright/playwright.config.ts'",
+        "test-ci": "PW_HEADLESS=true npm test",
+        "test-slomo": "npm run test-slomo --prefix ../../../mattermost/e2e-tests/playwright -- --project=chrome --config='../../../mattermost-plugin-todo/e2e/playwright/playwright.config.ts",
+        "debug": "npm test -- --debug",
+        "show-report": "npm run show-report --prefix ../../../mattermost/e2e-tests/playwright",
+        "lint": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js  --ext ts . --quiet --cache",
+        "lint:fix": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js  --ext ts . --quiet --cache --fix",
+        "tsc": "tsc"
+    },
+    "devDependencies": {
+        "@types/express": "^4.17.17",
+        "@types/pg": "^8.6.6",
+        "@typescript-eslint/eslint-plugin": "~5.59.0",
+        "@typescript-eslint/parser": "~5.59.0",
+        "cross-env": "7.0.3",
+        "dotenv": "^16.0.3",
+        "eslint": "7.9.0",
+        "eslint-plugin-import": "2.22.0",
+        "eslint-plugin-react": "~7.32.2",
+        "eslint-plugin-react-hooks": "~4.6.0",
```

Comment:
I think we can remove these too eslint plugins since we're not doing anything with React in this e2e test package